node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -550,21 +545,17 @@ void doConverge(NodeAgentContext context) {
                     orchestrator.resume(context.hostname().value());
                     suspendedInOrchestrator = false;
                 }
-                break;
-            case provisioned:
-                nodeRepository.setNodeState(context.hostname().value(), NodeState.ready);
-                break;
-            case dirty:
+            }
+            case provisioned -> nodeRepository.setNodeState(context.hostname().value(), NodeState.ready);
+            case dirty -> {
                 removeContainerIfNeededUpdateContainerState(context, container);
                 context.log(logger, "State is " + node.state() + ", will delete application storage and mark node as ready");
                 credentialsMaintainers.forEach(maintainer -> maintainer.clearCredentials(context));
                 storageMaintainer.syncLogs(context, false);
                 storageMaintainer.archiveNodeStorage(context);
                 updateNodeRepoWithCurrentAttributes(context, Optional.empty());
                 nodeRepository.setNodeState(context.hostname().value(), NodeState.ready);
-                break;
-            default:
```

Comment:
this is useful if the state is e.g. deprovisioned

Reply:
Will fix, had to merge since the internal PR was merged.